front-end/src/pages/HomePage.tsx

```diff
@@ -1,11 +1,8 @@
 import axios from 'axios';
 import React, { useEffect, useState } from 'react';
-import ReactDOM from "react-dom"
-import { RouteComponentProps } from 'react-router-dom';
-import { Row, Col, Form, Input, Button } from 'reactstrap';
+import { Row, Col, Form, Input } from 'reactstrap';
 import PostList from '../components/PostList';
 import { Post } from '../types/Post';
-import AuthorList from "../components/AuthorList"
 interface userProps {
```

Comment:
can this be changed to just have an `Author` object? I can do it. It's a quick fix